addons/l10n_cl/views/report_invoice.xml

```diff
@@ -1,65 +1,247 @@
-              primary="True">
+        <!-- we remove the taxes column in boletas -->
+        <xpath expr="//th[@name='th_taxes']/span" position="replace">
+            <t t-if="is_tax">
+                <span>Taxes</span>
+            </t>
+        </xpath>
 
-        <xpath expr="//h2" position="attributes">
-            <attribute name="t-if">not o.l10n_latam_document_type_id.report_name</attribute>
+        <xpath expr="//span[@id='line_tax_ids']" position="replace">
+            <t t-if="is_tax">
+                <span t-esc="', '.join(map(lambda x: (x.description or x.name), line.tax_ids))" id="line_tax_ids"/>
+            </t>
         </xpath>
 
-        <xpath expr="//h2" position="after">
-            <h2 t-if="o.l10n_latam_document_type_id.report_name">
-                <span t-field="o.l10n_latam_document_type_id.report_name"/>
-                <span t-field="o.l10n_latam_document_number"/>
-            </h2>
+        <!-- remove payment term, this is added on information section -->
+        <p name="payment_term" position="replace"/>
+
+
+        <xpath expr="//span[@t-field='o.invoice_payment_ref']/../.." position="replace"/>
+
+        <div id="informations" position="replace">
+            <t t-call="l10n_cl.informations"/>
+        </div>
+
+        <!--  we remove the ml auto and also give more space to avoid multiple lines on tax detail -->
+        <xpath expr="//div[@id='total']/div" position="attributes">
+            <attribute name="t-attf-class">#{'col-6' if report_type != 'html' else 'col-sm-7 col-md-6'}</attribute>
+        </xpath>
+
+        <xpath expr="//div[@id='total']/div" position="before">
+            <div t-attf-class="#{'col-6' if report_type != 'html' else 'col-sm-7 col-md-6'}"/>
         </xpath>
 
+        <xpath expr="//div[@id='total']" position="after">
+            <div class="row">
+                <div name="stamp" class="col-4 text-center"/>
+                <div name="transferable-table" class="col-4"/>
+                <div name="transferable-legend" class="col-4 pull-right"/>
+            </div>
+        </xpath>
+
+    </template>
+
+    <template id="report_invoice_document_with_payments" inherit_id="account.report_invoice_document_with_payments" primary="True">
+
+        <!-- custom header and footer -->
+        <t t-set="o" position="after">
+            <t t-set="custom_header" t-value="'l10n_cl.custom_header'"/>
+            <t t-set="report_date" t-value="o.invoice_date"/>
+            <t t-set="report_number" t-value="int(o.l10n_latam_document_number)"/>
+            <t t-set="pre_printed_report" t-value="report_type == 'pdf'"/>
+            <t t-set="report_name" t-value="o.l10n_latam_document_type_id.name"/>
+            <t t-set="header_address" t-value="o.company_id.partner_id"/>
+            <t t-set="is_tax" t-value="o.l10n_latam_document_type_id.code not in ['39', '41', '110', '111', '112', '34']"/>
```

Comment:
We can set this in the report itself.  No need to do that here as we pass parameters.

Reply:
Where do you mean?

Reply:
ready